bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -370,6 +362,13 @@ void BCStateTran::initImpl(uint64_t maxNumOfRequiredStoredCheckpoints,
     ConcordAssertEQ(replicaForStateTransfer_, nullptr);
     ConcordAssertEQ(sizeOfReservedPage, config_.sizeOfReservedPage);
 
+    maxVBlockSize_ = calcMaxVBlockSize(numberOfRequiredReservedPages, config_.sizeOfReservedPage);
+    maxItemSize_ = calcMaxItemSize(config_.maxBlockSize, numberOfRequiredReservedPages, config_.sizeOfReservedPage);
+    maxNumOfChunksInAppBlock_ =
+        calcMaxNumOfChunksInBlock(maxItemSize_, config_.maxBlockSize, config_.maxChunkSize, false);
+    maxNumOfChunksInVBlock_ = calcMaxNumOfChunksInBlock(maxItemSize_, config_.maxBlockSize, config_.maxChunkSize, true);
+    buffer_.reset(new char[maxItemSize_]);
+
```

Comment:
Please add a log printing those values and numberOfRequiredReservedPages

Reply:
numberOfRequiredReservedPages is assigned to numberOfReservedPages_ which is printed to log in line 382.